### PR TITLE
more README cleanup and documentation refresh

### PR DESCRIPTION
**Pull request by shiftkey**
Related to #1368

This moves the "add a project" section out to it's own documentation and provides a better walkthrough for adding your project. We haven't really done a good job of explaining the steps and things to look for, so hopefully this means reviews become simpler for us.

Rendered links:

 - [`README.md`](https://github.com/shiftkey/up-for-grabs.net/blob/more-docs-cleanup/README.md)
 - [`docs/list-a-project.md`](https://github.com/shiftkey/up-for-grabs.net/blob/more-docs-cleanup/docs/list-a-project.md)

### CODE REVIEW

**Comment by ritwik12** [comment]
@shiftkey Thanks for merging this, just got a bit busy and missed it :)

**Comment by shiftkey** [comment]
@ritwik12 totally understandable - if you have any suggestions to improve it I'm happy to submit a follow-up PR

File: README.md [hunk 1]
```diff
@@ -8,46 +8,10 @@ Do you run or participate in an open-source project? Submit a Pull Request to ad
...
-
-If you'd like to use a generator to create the project's file, you can certainly do so!
-
-Install the generator, then run it and walk through the steps.
-
-```
-npm install -g generator-up-for-grabs
-yo up-for-grabs
-```
+If you know of a project that should be listed on Up for Grabs,
```
Comment (ritwik12) [inline README.md:13]: `If you know of or own a project that should be listed on Up for Grabs`

File: docs/list-a-project.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+## Criteria
+
+We are interested in active open source projects that meet this criteria:
+
+- available maintainers who are interested in guiding new contributors
+- the project curates a list of small tasks which are ideal for new contributors
+- maintainers are happy and willing to review and work with new contributors to
+  help them learn more about open source
+
+If you project does not satisfy all of this criteria, it may be declined at the
```
Comment (ritwik12) [inline docs/list-a-project.md:15]: your

Comment (ritwik12) [reply docs/list-a-project.md:15]: Typo @shiftkey

**Comment by shiftkey** [comment]
@ritwik12 thanks! I've opened #1614 to get those changes in...

**Pull request merged**